#ifndef SettingSectionExtensionDialog_h
#define SettingSectionExtensionDialog_h


#include "ui_SettingFileAssociationExtensionDialog.h"

namespace CE{


class SettingFileAssociationExtensionDialog : public QDialog, public Ui::SettingFileAssociationExtensionDialog {
public:
    SettingFileAssociationExtensionDialog(QWidget *parent);
private:

};
}

#endif // SettingSectionExtensionItem_h
